packages/contracts/deployed-contracts.json

```diff
@@ -3805,5 +3805,11 @@
       "address": "0xe1B3b8F6b298b52bCd15357ED29e65e66a4045fF",
       "deployer": "0xc783df8a850f42e7F7e57013759C285caa701eB6"
     }
+  },
+  "GetAllTrancheAssetsData": {
```

Comment:
This is not needed. Pushing this file is kinda dumb. We need to have a production instance we push and a local instance that can change with any dev script. I'll push this for now and fix up this workflow in a separate pr